topology/Internet2/Internet2Denver/SITE.yaml

```diff
@@ -0,0 +1,5 @@
+ID: 10351
+LongName: Internet2 Denver
+Description: Backbone location of Internet2 at the Denver Point of Presence
+Latitude: 39.74582707084923
+Longitude: -104.97949077328207
```

Comment:
You're missing a newline at the end of file here (see bullet #2 here https://osg-htc.org/technology/software/git-software-development/#avoid-whitespace-noise)